homeassistant/components/velbus/__init__.py

```diff
@@ -105,10 +101,11 @@ def syn_clock(self, service=None):
 class VelbusEntity(Entity):
     """Representation of a Velbus entity."""
 
-    def __init__(self, module, channel):
+    def __init__(self, module, channel, counter=False):
```

Comment:
I'd expect to only modify the sensor entity and not the base entity. Make an init method in the sensor entity class and pass the remaining arguments to super.

Reply:
Fixed in this [commit](https://github.com/home-assistant/home-assistant/pull/31165/commits/33104b2cc59eeb8bcd497d9672f0dda7de8f1159)